tests/integration/__init__.py

```diff
@@ -332,3 +332,85 @@ def wait_for_down(self):
 
     def wait_for_up(self):
         self.up_event.wait()
+
+
+class BasicKeyspaceUnitTestCase(unittest.TestCase):
+    """
+    This is basic unit test case that can be leveraged to scope a keyspace to a specific unit test.
+    creates a keyspace named after the testclass with a rf of 1
+    """
+    @classmethod
+    def setUpClass(self):
+        self.commonSetUp('1')
```

Comment:
Minor nit, but it seems more natural to pass a number for rf and let the method take care of formatting.